pkg/webhooks/sgx/sgx.go

```diff
@@ -81,6 +81,21 @@ func getAesmdVolume(needsAesmd bool, epcUserCount int32, aesmdPresent bool) *cor
 	}
 }
 
+func warnWrongResources(resources map[string]int64) []string {
+	warnings := make([]string, 0)
+
+	_, ok := resources[encl]
+	if ok {
+		warnings = append(warnings, encl+" should not be used in Pod spec directly")
```

Comment:
Q: should we pass the container name here to specify which container in a pod triggers the warning?

Reply:
I think the caller should already know who made the request? Also the docs emphasize keeping the warnings as short as possible, 120 chars is the soft limit.

Reply:
I was thinking the case where, e.g., a pod has two containers and both are (wrongly) requesting enclave directly. Instead of seeing two warnings that look the same, the warning could be prefixed with the container name

Reply:
I'm good with either approach -- just pick the one you think better.